pkg/table/asserty/asserty.go

```diff
@@ -0,0 +1,68 @@
+// Package asserty offers functionality to assert for certain DB properties.
+package asserty
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+
+	"github.com/cashapp/spirit/pkg/table"
+	_ "github.com/go-sql-driver/mysql"
+)
+
+type Table struct {
+	ti *table.TableInfo
+}
+
+func LoadTable(db *sql.DB, schema, tableName string) (*Table, error) {
+	ti := table.NewTableInfo(db, schema, tableName)
+	if err := ti.SetInfo(context.TODO()); err != nil {
+		return nil, err
+	}
+	return &Table{ti: ti}, nil
+}
+
+func exists(val string, collection []string) bool {
```

Comment:
Can't you [use](https://golangtutorial.dev/tips/golang-slice-contains-method/) `slices.contains(collection, val)`?